### ISSUE TITLE
wrong default pick-up location for users in some databases

### ISSUE THREAD

**Issue by patrickzurek** [CARLI/vufind#13]
JIRA issue created by: Jessica Gibson (gibsonjc)
Originally opened: 2016-03-09 12:12 PM

It seems to be true of all MIL patrons that their default pick-up location is SAI in their account  And when I change it to the MIL location and click Save, it says it saves in a message, but then it immediately reverts back to SAI before my eyes.

I need to check more users in more databases.

**Comment by patrickzurek** [comment]
JIRA User: Jessica Gibson (gibsonjc)
JIRA Timestamp: 2016-03-24 09:05 AM

Today I can't find more evidence of this happening, so I'm going to resolve this issue.  I had one user from BRA who when I logged in the pick-up loc was SXU, but the change remained when I changed it to BRA unlike before with my MIL patrons.  All other patrons I have checked (about 8 in different dbs) have had "Select one" as the default in My Account > Profile, which seems right to me.
